### ISSUE TITLE
Courseplay extended memory usage

### ISSUE THREAD

**Issue by Lantmanenfs** [Courseplay/courseplay#6647]
Coursplayer use Lot of Memory on computer.    On my like 5gb om 4X ! 16x map crash!

**Comment by schwiti6190** [comment]
@Lantmanenfs Sadly this isn't really helping. It's not so easy to fix, as there are basically two/three devs actively working in their free time in between real life responsibilities currently and a few PR request from diffrent users here and there. It's an open source project which depends on the support of the community, so if you or others want to help either by contributing directly with fixes or by testing this feel free to help :)
To summarize the above it would be really helpful, if you or the community can test different courseplay setups, like shovel mode, graintransport mode .. and check the memory usage for these. Then it might be possible to lighten the load on memory.
The developers are dependent on the feedback by all of you, as you are equal to the test teams of game developers, who make a living doing it :)

**Comment by Lantmanenfs** [comment]
Think if deleting Auto scanning of the fields makes it easier. Coursplayer works fine otherwise. Had a late fs15

**Comment by Tensuko** [comment]
For sure can't delete Auto Scanning.
This is als just a temporary thing that is working on save startup.
You can turn it off thou if you won't want it to scan on gamestart.

**Comment by pvaiko** [comment]
Course generation and pathfinding also needs a lot of memory, there are no plans in the near future to optimize that.
